build.gradle

```diff
@@ -61,6 +60,59 @@ sonar {
     }
 }
 
+def getFixAndFullVersion() {
```

Comment:
I would prefer we don't use the "def" keyword and instead specify the actual type (e.g. `String[]`) instead (just like we don't use the `var` keyword in Java).

Reply:
I believe in Groovy this is how you do it. Let me change it then.

Reply:
Also, note that this method returns multiple values. I use it to access the values directly. To change this I would have to return a string instead which inmho I don't think is very clean.

Reply:
`return [ foo, bar ]` in Groovy means you return List, right? Technically your "def" means the method returns Object, which could be anything. I am not a Groovy expert, but maybe a better solution would be to declare a class with the fields we need (and their types well specified), and then return an instance of this class instead of Object/List?